dhcp/Vagrantfile

```diff
@@ -0,0 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  
+  config.vm.box = "ubuntu/bionic64"
+  
+  config.vm.provider "virtualbox" do |vb| 
+	  vb.memory = "512"  
+  end
+
+  config.vm.network "public_network", ip: "192.168.1.40", netmask: "255.255.255.0", virtualbox_intnet: "eno1"
```

Comment:
Here you should use `NETWORK` from `settings.sh`, otherwise it will not be on the same network as the ltsp-server and ltsp-client.